docs/DefiningLabels.md

```diff
@@ -0,0 +1,111 @@
+# Defining Labels
+
+Unless mentioned otherwise, **labels are applied to issues only** (not PRs). 
+
+## Label groups
+
+While GitHub does not have the concept of label groups, we can simulate label groups using systematic label naming. 
+i.e. labels having the same prefix are considered part of a label group.
+
+There are two types of label groups: 
+ 
+* Exclusive groups : No more than one label from the group can be applied to an issue
+* Non-exclusive groups:  Multiple labels from a group can be applied to an issue
+
+**Common label groups**:
+
+* exclusive : `s.` status, `p.` priority, `c.` category, `d.` difficulty, `e.` effort, 
+* non-exclusive:  `a-` aspect, `f-` feature, `t-` tech, 
+ 
+# Common Labels
+
+**Status (`s.`)**
+
+Status labels can be applied to PRs as well.
+
+* `s.OnHold` : The work on the PR has been put on hold pending some other event.
```

Comment:
> The new 'review status' lowers the need for status tables

I have trouble finding out how I can ask GitHub to just show me "PRs that I have to review" or "PRs I am a reviewer of" (The "PRs I am mentioned in" is useless because I'm mentioned quite a lot of times, and GitHub does not have a way of removing a PR from that list once I'm mentioned even once :-/)

Maybe I'm just bad at searching, but if GitHub doesn't have this feature then the "reviewer status" feature is quite useless and I'd much prefer assigning myself to the PR so that I can "bookmark" and monitor it.